wiki/Medals/zh.md

```diff
@@ -0,0 +1,127 @@
+谱面的星级将取整（如 5.88 星的图将会计为 5 星），并将给予玩家对应的奖章。
+使用难度增加或难度减少的 mods 来完成谱面或取得 S 评价也会计入奖章获取条件。
+注意，必须玩 Ranked 谱面才能获取奖章。
+以下展示的是8星的 _Pass_ 和 _S_ 奖章，实则从1星到8星都有对应的奖章。
+
+
+| osu!standard | osu!taiko | osu!catch | osu!mania |
+| :----------: | :-------: | :-------: | :-------: |
+| ![通过 osu!standard 8星图](img/osu-skill-pass-8.png "8 Star osu!standard Pass") | ![通过 osu!taiko 8星图](img/taiko-skill-pass-8.png "8 Star osu!taiko Pass") | ![通过 osu!catch 8星图](img/fruits-skill-pass-8.png "8 Star osu!catch Pass") | ![通过 osu!mania 8星图](img/mania-skill-pass-8.png "8 Star osu!mania Pass") |
+| ![S评价通过 osu!standard 8星图](img/osu-skill-fc-8.png "8 Star osu!standard S") | ![S评价通过 osu!taiko 8星图](img/taiko-skill-fc-8.png "8 Star osu!taiko S") | ![S评价通过 osu!catch 8星图](img/fruits-skill-fc-8.png "8 Star osu!catch S") | ![S评价通过 osu!mania 8星图](img/mania-skill-fc-8.png "8 Star osu!mania S") |
+
+毅力
+----------
+
+毅力奖章会随着游戏的深入而自然获得。尝试在 osu!standard 模式中完成更多的谱面，在 osu!catch 中接到更多的水果，在 osu!taiko 中击打更多次鼓面，在 osu!mania 中按更多的键数，就可以得到相应的奖章！
+
+| osu!standard | osu!taiko | osu!catch | osu!mania |
+| :----------: | :-------: | :-------: | :-------: |
+| ![玩 5,000 次(osu!standard)](img/osu-plays-5000.png "5,000 Plays (osu!standard mode)") | ![击 30,000 次鼓](img/taiko-hits-30000.png "30,000 Drum Hits") | ![接 20,000 个水果](img/fruits-hits-20000.png "Catch 20,000 fruits") | ![按 40,000 个键](img/mania-hits-40000.png "40,000 Keys") |
+| ![玩 15,000 次(osu!standard)](img/osu-plays-15000.png "15,000 Plays (osu!standard mode)") | ![击 300,000 次鼓](img/taiko-hits-300000.png "300,000 Drum Hits") | ![接 200,000 个水果](img/fruits-hits-200000.png "Catch 200,000 fruits") | ![按 400,000 个键](img/mania-hits-400000.png "400,000 Keys") |
+| ![玩 25,000 次(osu!standard)](img/osu-plays-25000.png "25,000 Plays (osu!standard mode)") | ![击 3,000,000 次鼓](img/taiko-hits-3000000.png "3,000,000 Drum Hits") | ![接 2,000,000 个水果](img/fruits-hits-2000000.png "Catch 2,000,000 fruits") | ![按 4,000,000 个键](img/mania-hits-4000000.png "4,000,000 Keys") |
+| ![玩50,000次(osu!standard)](img/osu-plays-50000.png "50,000 Plays (osu!standard mode)") | ![击 30,000,000 次鼓](img/taiko-hits-30000000.png "30,000,000 Drum Hits") | ![接 20,000,000 个水果](img/fruits-hits-20000000.png "Catch 20,000,000 fruits") | ![按 40,000,000 个键](img/mania-hits-40000000.png "40,000,000 Keys") |
+
+Mod 初体验
+-------------
+
+_参见：[Mods](/wiki/Game_Modifiers)。_
+
+mod 初体验类奖章可以通过初次使用相应的 mods 来完成谱面获得，以此来激励玩家主动尝试和体验游戏提供的各种不同的 mods。
+
+特别注意，如果在一次游玩中使用了2个或更多的 mods ，那么**不会**给玩家带来 _任何奖章_ 。
+ 
+
+|提高难度| 降低难度 | 特殊 |
+|:---:|:---:|:---:|
+|![Hard Rock](img/all-intro-hardrock.png "Hard Rock") | ![Hard Rock](img/all-intro-hardrock.png "Hard Rock")| ![Hard Rock](img/all-intro-hardrock.png "Hard Rock")|![Auto Pilot](img/all-intro-autopilot.png "Auto Pilot") |
+| ![Double Time](img/all-intro-doubletime.png "Double Time") | ![Nightcore](img/all-intro-nightcore.png "Nightcore") | ![Half Time](img/all-intro-halftime.png "Half Time") | ![Relax](img/all-intro-relax.png "Relax") |
```

Comment:
remove the **|** between the DT and NC medal, they belong together also it fucked up the table